apps/dashboard/app/models/motd_formatter_markdown_erb.rb

```diff
@@ -0,0 +1,14 @@
+class MotdFormatterMarkdownErb
+
+  attr_reader :content, :title
+
+  def initialize(motd_file)
+    motd_file = MotdFile.new unless motd_file
+    @title = motd_file.title
+    @content = OodAppkit.markdown.render(ERB.new(motd_file.content).result)

```

Comment:
We'll probably need to add rescues here given that templating could fail when things are undefined, but also it could throw an exception if say, the ERB tries to execute commands and _they_ fail.